functions/onprem/orborus/orborus.go

```diff
@@ -1150,6 +1152,18 @@ func deployWorker(image string, identifier string, env []string, executionReques
 		},
 		Resources: container.Resources{},
 	}
+    
+    _, err := os.ReadDir(certPath)
+
+    if certPath != "" && err == nil {
+        certVol := mount.Mount{
+            Type: mount.TypeBind,
+            Source: certPath,
+            Target: "/certs",
```

Comment:
So is shuffle-shared directly reading /certs every time? Did you try with/without root for this? 

Also, shouldn't Target be `certPath` and not Source? I'm a bit confused by the logic of this and how we control mounts based on it

Reply:
Yup you're write as this was still under draft, I was testing it locally with my custom `certPath` but yes it should be `/certs` and the target should also be `/certs` because I don't think it's a good idea to give user some independence on how they want to mount he volume internally.